pkg/sql/sem/tree/stmt.go

```diff
@@ -568,6 +568,12 @@ func (*Prepare) StatementType() StatementType { return Ack }
 // StatementTag returns a short string identifying the type of statement.
 func (*Prepare) StatementTag() string { return "PREPARE" }
 
+// StatementType implements the Statement interface.
+func (*ReassignOwned) StatementType() StatementType { return DDL } // TODO(angelaw): Is this correct?
```

Comment:
Again: I believe 'Priv' is not an option here so 'DDL' is the most appropriate return type. Would appreciate any thoughts/clarifications on the statement type however.

Reply:
Yeah it's technically a DDL. I think `Priv` is just a docs thing in sql.y for the generated bnf statements.